recipe/download_osx_sdk.sh

```diff
@@ -7,15 +7,12 @@ export MACOSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET:-10.9}
 # Some project require a new SDK version even though they can target older versions
 if [ -f ${CI_SUPPORT}/${CONFIG}.yaml ]; then
     export MACOSX_SDK_VERSION=$(cat ${CI_SUPPORT}/${CONFIG}.yaml | shyaml get-value MACOSX_SDK_VERSION.0 0)
-    export WITH_LATEST_OSX_SDK=$(cat ${CI_SUPPORT}/${CONFIG}.yaml | shyaml get-value WITH_LATEST_OSX_SDK.0 0)
-    if [[ "${WITH_LATEST_OSX_SDK}" != "0" ]]; then
-        echo "Setting WITH_LATEST_OSX_SDK is removed. Use MACOSX_SDK_VERSION to specify an explicit version for the SDK."
-        export MACOSX_SDK_VERSION=10.15
-    fi
 fi
 
 if [[ "${MACOSX_SDK_VERSION:-0}" == "0" ]]; then
-    export MACOSX_SDK_VERSION=$MACOSX_DEPLOYMENT_TARGET
+    # ensure minimal SDK of 10.13, while treatement of
+    # deployment targets <10.13 is being figured out
+    export MACOSX_SDK_VERSION="10.13"
```

Comment:
This would break when `MACOSX_DEPLOYMENT_TARGET>10.13`

Reply:
That branch is only taken if the value hasn't been populated.

Reply:
What value?

Reply:
MACOSX_SDK_VERSION. But I think I understand what you mean now. It would break setting `MACOSX_DEPLOYMENT_TARGET>10.13` without _also_ setting `MACOSX_SDK_VERSION>10.13`.

If we want to support the use-case of specifying the target without the sdk, I could rewrite this. I wasn't aware this was even possible and considered it a non-issue.